### PR TITLE
Helps 123 refactor self test loop

### PR DESCRIPTION
**Pull request by RusGraf**
This is the final selfTestLoop improvement for this run. We probably will come back to it and identify more improvements. Perhaps User Interactions automation ;) I renamed all tests to follow agreed convention and restructured couple tests which didnt make sense any more.

### CODE REVIEW

File: ldk/javascript/examples/self-test-loop/src/config/clipboard-group.ts [hunk 1]
```diff
@@ -7,13 +7,13 @@ export const clipboardTestGroup = (): TestGroup =>
...
+      clipboardTests.testWriteAndRead,
       10000,
       'Copying value to clipboard and reading it back',
     ),
     new LoopTest(
       'Clipboard Aptitude - Clipboard Stream',
-      clipboardTests.clipboardStream,
+      clipboardTests.testListen,
       10000,
       'Copying the value "LDKThxBai" the the clipboard',
```
Comment (MaggieMaSS) [inline ldk/javascript/examples/self-test-loop/src/config/clipboard-group.ts:18]: Well I know you didn't change this tho, but it should be ```Copying the value "LDKThxBai" to the clipboard'```

Comment (RusGraf) [reply ldk/javascript/examples/self-test-loop/src/config/clipboard-group.ts:18]: Ahh good catch

File: ldk/javascript/examples/self-test-loop/src/tests/filesystem/index.ts [hunk 2]
```diff
@@ -35,128 +36,116 @@ export const queryDirectory = (): Promise<boolean> =>
...
+      .encode(testString)
+      .then((encodedValue) => {
+        filesystem
+          .writeFile({
+            path: filePath,
+            data: encodedValue,
+            writeOperation: filesystem.WriteOperation.overwrite,
+            writeMode,
+          })
+          .then(() => {
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/tests/filesystem/index.ts:102]: i see you eradicated some of the `.then`s, but not others. do we have plans to remove them all?

File: ldk/javascript/examples/self-test-loop/src/tests/whisper/whisper-update.ts [hunk 3]
```diff
@@ -282,11 +283,11 @@ export const updateOnChange = (): Promise<boolean> =>
                       },
                     ],
                   },
-                  (error) => {
```
Comment (cshields-oliveai) [inline ldk/javascript/examples/self-test-loop/src/tests/whisper/whisper-update.ts:285]: what do you have against `error` :)

**Review by cshields-oliveai — APPROVED** [review]

File: ldk/javascript/examples/self-test-loop/src/tests/filesystem/index.ts (hunk 2 above)
Comment (RusGraf) [reply ldk/javascript/examples/self-test-loop/src/tests/filesystem/index.ts:102]: I could sweep through a whole thing. Initially I tried to keep changes to a min to reduce conflicts. But had to touch lots of things with timeout removal and took care of everything what I touched. We are close, perhaps we will improve while we go.

**Review by epiccoleman — APPROVED** [review]

**Review by BrettMugglinOlive — APPROVED** [review]

**Pull request merged**